firmware/src/core/wifiWidget.cpp

```diff
@@ -34,9 +35,13 @@ void WifiWidget::setup() {
     wifimgr.setShowInfoErase(false);
     wifimgr.setMenu(wm_menu);
   
-    // reset settings - wipe stored credentials for testing
+    // Hold right button when connecting to power to reset wifi settings
     // these are stored by the ESP WiFi library
-    //wifimgr.resetSettings();
+    if (digitalRead(BUTTON_RIGHT) == Button::PRESSED_LEVEL) {
+      wifimgr.resetSettings();
+      m_manager.drawCentreString("Wifi Settings reset", ScreenCenterX, ScreenCenterY + lineHeight, fontSize);
+      delay(messageDelay);
+    }
```

Comment:
Changes like this do not belong in this PR imo.

Reply:
Altough you are technically right, I could not try the WifiWidgets TTFs without it and I just kept it in.
It's such a small change that a separate PR is not really necessary IMHO.